### PR TITLE
feat(column): functionality to convert column values

### PR DESCRIPTION
**Pull request by VMBindraban**
Tried using the logic of aurelia to determine the converter and arguments but the closest thing i found was https://github.com/aurelia/binding/blob/master/src/parser.js#L37

### CODE REVIEW

File: doc/usage.md [hunk 1]
```diff
@@ -38,6 +38,27 @@ Comma-separated string representing the column names to display. This is used fo
...
+    .standardConfiguration()
+    .globalResources('dateFormatValueConverter')
+    .globalResources('priceFormatterValueConverter')
+}
+```
+
+```html
+<datatable 
+  resource="product" 
+  columns="name, price as 'Market price' | , createdAt | dateFormat: 'yyyy-mm-dd'"
```
Comment (doktordirk) [inline doc/usage.md]: priceValue missing?

File: package.json [hunk 2]
```diff
@@ -37,22 +37,33 @@
...
+      "aurelia-view-manager": "^0.0.4",
+      "json-statham": "^1.3.0",
+      "typer": "^1.1.0"
     },
     "devDependencies": {
       "aurelia-pal-browser": "^1.0.0-rc.1.0.0",
       "aurelia-polyfills": "^1.0.0-rc.1.0.0",
+      "babel": "babel-core@^5.8.24",
+      "babel-runtime": "^5.8.24",
+      "core-js": "^1.1.4",
```
Comment (doktordirk) [inline package.json]: why babel again? please remove

File: config.js [hunk 3]
```diff
@@ -1,28 +1,65 @@
 System.config({
   baseURL: "/",
   defaultJSExtensions: true,
-  transpiler: false,
+  transpiler: "babel",
+  babelOptions: {
+    "optional": [
+      "runtime",
+      "optimisation.modules.system"
+    ]
```
Comment (doktordirk) [inline config.js]: babel not needed. please set back to false

File: doc/usage.md [hunk 4]
```diff
@@ -38,6 +38,27 @@ Comma-separated string representing the column names to display. This is used fo
 <datatable resource="product" populate="group" columns="name,price as 'Market price',group.name as 'Product group'"></datatable>
 ```
 
+#### valueConverters
+You can give every colum a `valueConverter`. You need to define the converter in your `main.js` as a global resource. *[More information](http://eisenbergeffect.bluespire.com/binding-with-value-converters-in-aurelia/)()
```
Comment (doktordirk) [inline doc/usage.md]: atm it.s a single converter, right?

**Comment by doktordirk** [comment]
i think the important thing is that the syntax is fixed.(which it is i guess). then we always can extend later if desired

**Comment by VMBindraban** [comment]
I am using the aurelia syntax so.. yeah

**Comment by doktordirk** [comment]
alright, works like charm it seems

**Pull request merged**